### ISSUE TITLE
[Question] Safari doesn't run during CI (Getting stuck) without log info

### ISSUE THREAD

**Issue by ValorHeart** [microsoft/playwright#20679]
**Context:**
- Playwright Version: 1.29.2 and 1.30.0
- Operating System: Mac
- Node.js version: 16.14.0
- Browser: WebKit
- Extra: CircleCI with playwright:1.29.2-jammy docker image (or 1.30.0)

**Describe the bug**
There is no log output which makes it hard for me to explain and supply additional info.
I'm running my tests in parallel machines using shards, Chrome and Firefox work just fine, but when the tests arrive to Webkit project, it get stuck without any type of error as shown in the image below:
<img width="1555" alt="image" src="https://user-images.githubusercontent.com/61405630/217038436-14ac289d-2591-433d-afd6-61ac283eec06.png">

On my local host WebKit works just fine.

What I've tried:
- Running on 1.29.2 and 1.30.0 - Same result, no log whatsoever.
- Manually entering the CircleCI container and run the command `CI=true npx playwright test tests/playwright/tests --config=tests/playwright --shard=3/4` - Same result, no log whatsoever.

I can't run debug and didn't find any type of verbose command to have more info.

**Is there a way to add additional information to the console?**

Thanks, Adi

**Comment by mxschmitt** [comment]
can you share your config with us? It looks like your testTimeout is too high, so CircleCI in the end just kills the process and Playwright can't generate a good helpful output. I'd recommend to reduce the test-timeout, then the test will timeout and show you where it was hanging and enable [Playwright Tracing / HTML Report](https://playwright.dev/docs/ci-intro#html-report) to find out how your site was behaving.

**Comment by ValorHeart** [comment]
Thanks, @mxschmitt !
So I've gone even further and created a new default project from `npm init playwright@latest` and the problem yet still occurred.
No matter how short my timeout was, the tests would fail on timeout without any additional info shown in the image below
<img width="2404" alt="image" src="https://user-images.githubusercontent.com/61405630/217281113-ad5dfdd9-bc4d-4be1-875f-8148cb2ff15b.png">

- When I have pulled `playwright:v1.30.0-jammy` docker image to my M1 based mac it worked 🤷‍♂️

Finally, after I've exhausted the investigation abilities I have with my knowledge about how things work here, I did try to change the ubuntu distribution from `playwright:v1.30.0-jammy` to `playwright:v1.30.0-focal` and everything started to work!

so the current consultation is unfortunately unknown, but the solution was to switch to `playwright:v1.30.0-jammy` docker image.

my CI system is an on-premise CircleCI version 4 server.
If someone else will face the same issue in the future, hopefully, he'll find that post a bit usefull.

Thanks, Adi.
